components/Sidebar/Sidebar.js

```diff
@@ -22,6 +22,9 @@ function Sidebar({ isOpen, toggle }) {
           <Link href={"/"} passHref>
             <SidebarLink onClick={toggle}>Home</SidebarLink>
           </Link>
+          <Link href={"/about"} passHref>
+            <SidebarLink onClick={toggle}>About</SidebarLink>
```

Comment:
I pref convention like `() => toggle()`. It's more clearly what is happening there but thats okay